docs/api/errors.rst

```diff
@@ -87,20 +87,107 @@ Mixins
 Predefined Errors
 -----------------
 
-.. automodule:: falcon
```

Comment:
This was preventing other docs from being able to create reference links to the error class docs.